src/v1/sources/shopify/webhookTransformations/serverSideTransform.js

```diff
@@ -111,11 +113,16 @@ const processEvent = async (inputEvent, metricMetadata) => {
   // attach anonymousId if the event is track event using note_attributes
   if (message.type !== EventType.IDENTIFY) {
     await setAnonymousId(message, event, metricMetadata);
+    await updateAnonymousIdToUserIdInRedis(message);
```

Comment:
message signature for `updateAnonymousIdToUserIdInRedis` is wrong

Reply:
thanks, updated